docs/guides/introduction-to-camunda-8.md

```diff
@@ -6,44 +6,92 @@ slug: /guides/
-- [Zeebe](/components/zeebe/zeebe-overview.md) - The cloud-native process engine of Camunda 8.
-- [Tasklist](/components/tasklist/introduction-to-tasklist.md) - Complete tasks which require human input.
-- [Operate](/components/operate/operate-introduction.md) - Manage, monitor, and troubleshoot your processes.
-- [Optimize]($optimize$/components/what-is-optimize) - Improve your processes by identifying constraints in your system.
+Additionally, Camunda hosts a section of [Best Practices](/components/best-practices/best-practices-overview.md). A mix of conceptual and practical implementation information, this section hosts our condensed experience using BPMN and DMN on the Camunda tool stack collected by consulting engagement with our customers and feedback from the community.
 
-For more conceptual information about Camunda 8, see [What is Camunda 8](components/concepts/what-is-camunda-8.md).
-For an on-demand demonstration of the product, visit [Camunda Academy](https://bit.ly/3CvooTX).
+:::note Looking for deployment guides?
+
+Deployment guides for Camunda 8 components are available in the [Self-Managed section]/self-managed/about-self-managed.md.
+
+:::
 
-:::note
+:::note Interested in migrating from Camunda 7 to Camunda 8?
 Interested in migrating process solutions developed for Camunda 7 to run them on Camunda 8? Visit our guide on [migrating from Camunda 7](/guides/migrating-from-camunda-7/index.md).
 :::
 
 ## Getting started
 
 In this section of the Camunda 8 documentation, you'll find guides to get started with Camunda 8 before learning more about each individual [component](/components/components-overview.md). Our getting started guides walk you through the following steps:
 
-1. [Create a Camunda 8 account](./create-account.md) - Create a Camunda 8 account to create clusters, deploy processes, and create a new instance.
-2. [Model your first process](./model-your-first-process.md) - Design and deploy a process, and share the process with your teammates or other stakeholders to begin collaborating.
-3. [Create a cluster](./create-cluster.md) - Create your first cluster to execute processes.
-4. [Orchestrate human tasks](./getting-started-orchestrate-human-tasks.md) - Assign human tasks to users so they can enter the necessary data to drive the business process forward.
-5. [Get started with API orchestration](./getting-started-orchestrate-apis.md) - Configure Connector tasks with domain-specific parameters.
-6. [Orchestrate microservices](./getting-started-orchestrate-microservices.md) - Orchestrate the microservices necessary to achieve your end-to-end automated business process.
+<DocCardList items={[{type:"link", href:"/docs/next/guides/getting-started/", label: "Step 1: Create a Camunda 8 account", docId:"guides/getting-started"},
```

Comment:
```suggestion
<DocCardList items={[{type:"link", href:"/docs/next/guides/create-account/", label: "Step 1: Create a Camunda 8 account", docId:"guides/create-account"},
```

`guides/getting-started/` is the slug for this page. I think you want the page id in both places.